fhir-config/pom.xml

```diff
@@ -17,19 +17,14 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>jakarta.json</groupId>
-            <artifactId>jakarta.json-api</artifactId>
+            <groupId>org.glassfish</groupId>
+            <artifactId>jakarta.json</artifactId>
```

Comment:
Have you tried a build with java9? 

per http://repo1.maven.org/maven2/org/glassfish/json/1.1.6/json-1.1.6.pom

<activation>
<jdk>[9,)</jdk>
</activation>

You'll end up with lots of modules. 
<module>api</module>
<module>impl</module>
<module>jaxrs</module>
<module>jaxrs-1x</module>
<module>tests</module>
<module>gf</module>
<module>demos</module>
<module>bundles</module>

it's worth double checking instead of embedding the 'extra' jars everytime.

Reply:
I'm building with openjdk 11.0.5, but with a target of Java 8 thanks to the maven-compile-plugin.

Anyway, I think those extra dependencies you're looking at are all in the `dependencyManagement` section.

```
[INFO] --- maven-dependency-plugin:2.8:tree (default-cli) @ fhir-config ---
[INFO] com.ibm.fhir:fhir-config:jar:4.0.0-SNAPSHOT
[INFO] +- com.ibm.fhir:fhir-core:jar:4.0.0-SNAPSHOT:compile
[INFO] +- org.glassfish:jakarta.json:jar:1.1.5:compile
[INFO] +- org.testng:testng:jar:6.14.3:test
[INFO] |  +- com.beust:jcommander:jar:1.72:test
[INFO] |  \- org.apache-extras.beanshell:bsh:jar:2.0b6:test
[INFO] \- com.google.code.gson:gson:jar:2.8.1:compile
```

Reply:
never mind, now i see what you're talking about.  there's a profile section in their pom:
```
<profile>
  <id>jdk9-all</id>
  <activation>
    <jdk>[9,)</jdk>
  </activation>
  <modules>
    <module>api</module>
    <module>impl</module>
    <module>jaxrs</module>
    <module>jaxrs-1x</module>
    <module>tests</module>
    <module>gf</module>
    <module>demos</module>
    <module>bundles</module>
  </modules>
</profile>
```

still, i'm not sure how it relates to our usage.  we already have quite a few 'compile' dependencies on this module and i don't see any ways of asking for specific sub-modules from maven.

Reply:
it's active with java9.  just worth us thinking about...